tools/check-nuttx-release.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+#############################################################################
+# tools/check-nuttx-release.sh
```

Comment:
How about change the script name to checkrelease.sh like checkpatch.sh?

Reply:
I actually thought about this too.  I'm glad someone else had the same idea.

Reply:
@xiaoxiang781216 @Ouss4 I will change the script name